src/main/java/com/redhat/cloud/notifications/GwResource.java

```diff
@@ -73,6 +74,13 @@ public GwResource(MeterRegistry registry) {
     public Response forward(@NotNull @Valid RestAction ra) {
         receivedActions.increment();
 
+        try {
+            restValidationClient.validate(ra.getBundle(), ra.getApplication(), ra.getEventType());
+        } catch (BadRequestException e) {
+            // The following line is required to forward the HTTP 400 error message.
+            return Response.status(BAD_REQUEST).entity(e.getMessage()).build();
```

Comment:
Isn't this automatically forwarded by quarkus if not catched? (the `BadRequest`)

Reply:
The status is forwarded but the message is not, so this is necessary. We have the same thing in notifications.